systems/WikibaseClient/05-Building_Block_View.md

```diff
@@ -3,3 +3,61 @@
 ## Whitebox Overall System
 
 ![Alt Text](./diagrams/05-building-blocks.drawio.svg)
+
+## User Interfaces
+
+## APIs
+
+### Action API Description
+
+![Alt Text](./diagrams/05-api-description.drawio.svg)
+
+|  Building Block | Type/Context  | Responsibility   |
+| ------------ | ------------ | ------------ |
+| DescriptionLookup   | Lookups | Lookup client descriptions from a variety of sources |
+| RepoLinker   | Response | Creates links to Repo Entity concepts |
+
+### Action API EntityUsage
```

Comment:
What do you think about linking to the API description from these headlines? They usually contain a short description of what they do which might be useful.

```suggestion
### [Action API EntityUsage](https://www.wikidata.org/w/api.php?action=help&modules=query%2Bwbentityusage)
```

Reply:
I wonder if this should be a heading link, or in some other information bar?
I guess lets try the heading link for now?